GraphBulkImporter/App.config

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
+    </startup>
+  <appSettings>
+    <add key="EndPointUrl" value="https://localhost:8081/" />
+    <add key="AuthorizationKey" value="C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==" />
```

Comment:
I am guessing this shouldnt be checked in?

Reply:
Slow cheetah is your friend here. Let's you add transforms to your config file. You can put your key in a transform file and not check it in. https://github.com/Microsoft/slow-cheetah

Reply:
well, yes, you are correct. but in this case it doesn't actually matter b/c that's the key for the local emulator. it's a static key which is the same for everyone. the product team even have it in their docs on https://docs.microsoft.com :)